web/gui-v2/src/components/StatWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
I don't remember the details of the almanac statbox, other than that the styling is a little different but it's a broadly similar component. Do you think that's something that makes sense to refactor out into the shared components library? (I'm genuinely unsure - while this may be a kind of component we use in a number of tools, making a shared component sufficiently general to support all the tweaks that might be needed for an individual app might not be worth the effort considering that this is pretty simple to implement)

Reply:
This StatWrapper is really simple, but the one in Almanac has a bunch of various options, so I think at this point refactoring them out into UI Components would cause more complexity than its worth.

Reply:
👍